node/src/cli.rs

```diff
@@ -109,3 +112,42 @@ impl RelayChainCli {
         }
     }
 }
+
+#[derive(Debug, clap::Parser)]
+#[group(skip)]
+pub struct RunCmd {
+    #[clap(flatten)]
+    pub base: cumulus_client_cli::RunCmd,
+
+    /// P2P ED25519 private key
```

Comment:
P2P is too generic, I'd say something like:
```suggestion
    /// Key identifying nodes in p2p network of Storage Providers and Collators.
    /// It must be ED25519 private key, supplied as a short format XXXX or XXX or XXX.
```

(please fill the description of how it should look like)